bitnami/keycloak/templates/statefulset.yaml

```
@@ -59,9 +59,45 @@ spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
-      {{- if .Values.initContainers }}
+      {{- if or .Values.initContainers .Values.auth.tls.enabled }}
       initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.auth.tls.enabled }}
+      {{- $fullname := include "keycloak.fullname" . }}
+        - name: init-certs
+          image: {{ include "keycloak.auth.tls.image" . }}
+          imagePullPolicy: {{ .Values.auth.tls.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          command:
+          - /bin/sh
+          - -c
```

Our `bitnami/minideb` image gives some errors if we use `sh`, please change it to `bin/bash`:

```suggestion
          - /bin/bash
          - -ec
```